packages/accounts-controller/src/AccountsController.ts

```diff
@@ -362,19 +360,53 @@ export class AccountsController extends BaseControllerV2<
    */
   loadBackup(backup: AccountsControllerState): void {
     if (backup.internalAccounts) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore Type instantiation is excessively deep and possibly infinite.
       this.update((currentState: AccountsControllerState) => {
         currentState.internalAccounts = backup.internalAccounts;
       });
     }
   }
 
+  /**
+   * Generates an internal account for a non-Snap account.
+   * @param address - The address of the account.
+   * @param type - The type of the account.
+   * @returns The generated internal account.
+   */
+  #generateInternalAccountForNonSnapAccount(
+    address: string,
+    type: string,
+  ): InternalAccount {
+    return {
+      id: getUUIDFromAddressOfNormalAccount(address),
+      address,
+      options: {},
+      methods: [
+        'personal_sign',
+        'eth_sign',
+        'eth_signTransaction',
+        'eth_signTypedData_v1',
+        'eth_signTypedData_v3',
+        'eth_signTypedData_v4',
+      ],
```

Comment:
I see you're passing this value directly, what if one of the keyrings does not support one or more methods?

Reply:
These keyrings will always support all the methods, they are regular EOA accounts